nengo_gui/static/components/netgraph_item.js

```diff
@@ -356,8 +356,11 @@ Nengo.NetGraphItem.prototype.generate_menu = function () {
         }
     }
     if (this.sp_targets.length > 0) {
-        items.push(['Semantic pointer',
+        // Naming things is hard
+        items.push(['Semantic pointer similarity text',
```

Comment:
How about something like 'Semantic pointer cloud' (the plot reminds me somewhat of those tag clouds you see sometimes on blogs).